src/index.ts

```diff
@@ -364,13 +365,12 @@ export class GalleryComponent extends BaseComponent {
               data-initialwidth="${initialWidth}" 
               data-initialheight="${initialHeight}">
         <div class="wrap" 
-             style="width:${initialWidth}px; height:${initialHeight}px" 
-             class="${multiSelected ? 'multiSelected' : ''}">
+             style="width:${initialWidth}px; height:${initialHeight}px">
           ${multiSelectEnabled ? `
-          <input id="thumb-checkbox-${thumbId}" 
+          <input id="thumb-checkbox-${index}" 

```

Comment:
This was actually broken on master; it was using thumbId which is not defined. Didn't seem to be a problem until now where I needed to use this input id as a selector.